index.html

```diff
@@ -398,68 +398,37 @@ <h2>GPC Support Representation</h2>
     <section class="informative">
       <h2>Legal Effects</h2>
       <p>
-        Receiving a GPC signal may have legal effects, depending on
-        factors such as the location of the individual sending the signal, the scope of the
-        applicable law, as well as any separate agreement between the recipient of the signal and
-        the individual. For additional details on legal effects, 
+        The GPC signal was designed to allow users to take advantage of legal rights to stop certain
+        sharing or processing of their data. As such, the sending and receipt of a GPC signal may
+        have legal effects, depending on factors such as the location of the individual sending the
+        signal, the scope of the applicable law, as well as any separate agreement between the
+        recipient of the signal and the individual. For additional details on legal effects, 
         <a href="https://privacycg.github.io/gpc-spec/explainer" target="_blank">consult the explainer</a>. 
       </p>
       <p>
         For example, the use of the GPC signal by an individual will be intended to communicate the
         individual's intention to invoke the following rights, as applicable:
       </p>
-      <h3>Calfornia Consumer Privacy Act (CCPA)</h3>
+      <h3>United States Privacy Law</h3>
```

Comment:
I suggest stopping here, as the following text still speculates on how states may interpret GPC signals, which fits better into the explainer document.

Reply:
I think there's some value in having some very basic bare bones information about legal consequences in the spec without forcing people to go hunt down another document. I think everything about US law is factual and non-controversial. The only thing vaguely speculative is that states without rulemaking could try to provide guidance in other ways. I don't feel strongly about keeping that, I was just trying to provide basic info to folks, and then if states DO provide that information guidance, that would be in the explainer.

The stuff later on about other jurisdictions is more speculative, I would be ok moving that to the explainer, or just stripping it down to say application elsewhere around the world isn't currently clear.

Reply:
I agree, so I'll review again once this is merged and may raise another PR to suggest follow up changes.